pkg/cmd/cli/cmd/newapp.go

```diff
@@ -166,6 +166,8 @@ func RunNewApplication(fullName string, f *clientcmd.Factory, out io.Writer, c *
 			}
 		}
 	}
+	fmt.Fprintln(c.Out(), "Run 'oc status' to view your app.")
```

Comment:
> Isn't it better to use the fullName var instead of explicitly specifying oc?

Not that it's a great excuse, but nothing else in `new-app` is doing that and I'm actually not sure what I'd like to do about it.  Constants with the expected fullNames of commands occur to me since it doesn't change often and having to make a code change to have it match doesn't much bother me, but it's less than ideal.